packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/container.test.tsx

```diff
@@ -28,10 +36,58 @@ jest.mock('../../../models', () => {
   };
 });
 
+const mockNavigate = jest.fn();
+jest.mock('@reach/router', () => ({
+  ...jest.requireActual('@reach/router'),
+  useNavigate: () => mockNavigate,
+}));
+
 // Global instances
 let integration: Integration;
 let mockAuthClient = new AuthClient('localhost:9000');
 let currentProps: any | undefined;
+let mockEmailBounceStatusQuery = jest.fn();
+
+function mockLocation(
+  originIsSignup: boolean = true,
+  withIntegrationProps: boolean = true
+) {
+  const initialParams = new URLSearchParams({
+    email: MOCK_EMAIL,
+    emailFromContent: 'true',
```

Comment:
Mentioned in Slack but so it doesn't get lost, we remove these since we're not passing them from `router.js` anymore and we don't need to import `URLSearchParams`.